source/docs/hardware/hardware-tutorials/transmissions.rst

```diff
@@ -0,0 +1,7 @@
+Power Transmission
+==================
+Hand in hand with choosing the correct motors for an application is transmitting that motor power to place its needed. Using gears or chains and sprockets are two effective ways of matching the motor power to the application being driven. In this video WPI Robotics Engineering PhD student Michael Delph talks about power transmission including choosing correct gear or chain and sprocket ratios to get the the maximum performance from your robot design.
```

Comment:
```suggestion
Hand in hand with choosing the correct motors for an application is transmitting that motor power to the place it's needed. Using gears or chains and sprockets are two effective ways of matching the motor power to the application being driven. In this video, WPI Robotics Engineering PhD student Michael Delph talks about power transmission, including choosing correct gear or chain and sprocket ratios to get the the maximum performance from your robot design.
```
- Add the
- its (possessive) -> it's (it is)
- Comma after introductory phrase
- Comma before non-restrictive phrase